modules/monitoring/templates/smokey-loop.conf

```diff
@@ -6,4 +6,4 @@ description "Smokey loop outputs JSON which is consumed by monitoring"
 start on runlevel [2345] and stopped $UPSTART_JOB
 respawn
 
-exec /opt/smokey/tests_json_output.sh /tmp/smokey.json
```

Comment:
Had a look myself for similar things, and it seems like its all explicit in hiera for each class that needs to know, so I've done the same now for smokey  	9683414.